bridge-stp.in

```diff
@@ -212,7 +212,7 @@ case "$action" in
             # Add bridge to mstpd and configure.
             echo "Adding/configuring bridge '$bridge' ..."
             "$mstpctl" addbridge "$bridge" || continue
-            if [ -n "$config_cmd" ]; then
+            if [ -x "$config_cmd" ]; then
```

Comment:
@dvlemplgk ?
thoughts ?
if you agree, feel free to re-spin

also, i forgot to mention ; feel free to propose another Nth option to vote upon, if you disagree with any option I proposed